go/vt/mysqlctl/compression_benchmark_test.go

```diff
@@ -0,0 +1,190 @@
+package mysqlctl
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"vitess.io/vitess/go/vt/logutil"
+)
+
+type (
+	benchmarkCompressArgs struct {
+		b        *testing.B
+		builtin  string
+		external string
+	}
+
+	benchmarkCompressEnv struct {
+		benchmarkCompressArgs
+	}
+)
+
+const (
+	usage = `
+This benchmark tests the performance of built-in and external compressors,
+using mysqlctl.newBuiltinDecompressor and mysqlctl.newExternalCompressor
+to obtain compressors.
+
+You must supply a file to compress by setting this environment variable:
+
+    export VT_MYSQLCTL_COMPRESSION_BENCHMARK_DATA_PATH=path/to/data.xml
+
+We recommend using a data set that compresses well, such as:
+
+ - The enwik9 dataset: http://mattmahoney.net/dc/textdata
+ - A larger Wikipedia dataset from: https://en.wikipedia.org/wiki/Wikipedia:Database_download
```

Comment:
or we can use dd (https://www.cyberciti.biz/faq/howto-create-lage-files-with-dd-command/)
Not sure if dd does not result in too much compression..

Reply:
I can add it if you think it makes sense. I initially tried `head -c[bignumber] /dev/urandom`, but ended up with too little compression. Got guidance to use realistic datasets.

Reply:
make sense.. I think i like wikipedia databset.. it looked more realistic.